tools/submission/generate-final-report.py

```diff
@@ -55,7 +55,7 @@ def main():
       axis=1)
 
   # details url
-  base_url = 'https://github.com/mlcommons/submissions_inference_1_1/tree/master'
+  base_url = 'https://github.com/mlcommons/submissions_inference_2_0/tree/master'
```

Comment:
The v2.0 submission tree is https://github.com/mlcommons/submissions_inference_2.0/tree/master (ending with `2.0`, not `2_0`).